bin/package/installation/post-install.sh

```diff
@@ -39,6 +39,28 @@ function install_chkconfig {
         && chkconfig --add mysterium-node
 }
 
+function ensure_paths {
+    iptables_path=`which iptables`
+    if [[ ${iptables_path} == "" ]]; then
+       echo "required dependency missing: iptables"
```

Comment:
It should be enough to have just dependency for the package, no need to recheck it in `post-install` script.

Reply:
Added this for cases, when user forces dpkg install. This lets install package without dependencies.

Reply:
Maybe you are right. Removed this additional check. User should rely on OS provided depencency policy.